packages/react/src/components/form/initial.tsx

```diff
@@ -225,6 +225,8 @@ export const Initial: React.FC<Props> = ({ flowState }) => {
     () => factors.filter(isFactorOidc),
     [factors]
   );
+  const hasOidcFactors = Boolean(oidcFactors.length);
```

Comment:
What if the factors only include the OIDC factors? Eg if we remove all other methods, would we still have this divider displayed?